packages/trader/src/AppV2/Components/Filter/contract-type-filter.tsx

```diff
@@ -27,24 +27,23 @@ const ContractTypeFilter = ({ contractTypeFilter, onApplyContractTypeFilter }: T
         }
     };
 
-    const getChipLabel = () => {
-        const arrayLength = contractTypeFilter.length;
-        if (!arrayLength) return <Localize i18n_default_text='All trade types' key='All trade types' />;
-        if (arrayLength === 1) return AVAILABLE_CONTRACTS.find(type => type.id === contractTypeFilter[0])?.tradeType;
-        return <Localize i18n_default_text='{{amount}} trade types' values={{ amount: arrayLength }} key='Amount' />;
-    };
-
     return (
         <React.Fragment>
             <Chip.Standard
                 className='filter__chip'
                 dropdown
                 isDropdownOpen={isDropdownOpen}
+                label='Trade types'
```

Comment:
Do we need to use Localize here?

Reply:
absolutely

Reply:
Please, check if it will cause new issues with keys in the test links. I noticed, that it's very common case, when we pass <Localize/> in quill and it complains about keys(

Reply:
yeah, it will complain, on it) no worries